google-cloud-firestore/src/test/java/com/google/cloud/firestore/MapperTest.java

```diff
@@ -1211,17 +1223,9 @@ public void primitiveDeserializeLong() {
     }
   }
 
-  @Test
+  @Test(expected = RuntimeException.class)
   public void primitiveDeserializeWrongTypeMap() {
-    assertExceptionContains(
-        "Failed to convert value of type java.util.LinkedHashMap to String "
-            + "(found in field 'value')",
-        new Runnable() {
-          @Override
-          public void run() {
-            deserialize("{'value': {'foo': 'bar'}}", StringBean.class);
-          }
-        });
+    deserialize("{'value': {'foo': 'bar'}}", StringBean.class);
```

Comment:
This test loses the exception text, which verifies that we tell the developer where the exception occurred.